products/PurePhone/BinaryAssetsVersions.cmake

```diff
@@ -1,7 +1,7 @@
 # This file sets versions of downloaded binaries for release packaging purposes
 
 if( NOT DEFINED ECOBOOT_BIN_VERSION)
-    set(ECOBOOT_BIN_VERSION 1.1.0 CACHE STRING "bootloader binary version to download from bootloader release page")
+    set(ECOBOOT_BIN_VERSION 1.1.2 CACHE STRING "bootloader binary version to download from bootloader release page")
```

Comment:
like I said - we have to bump it right after we release new bootloader or we'll forget about it :D